cmd/server.go

```diff
@@ -64,15 +64,13 @@ be displayed and one can be selected.
 }
 
 // serverCurrent command to display the name of the current server in the local config
-func serverCurrent(ctx *cli.Context) error {
-	cf, err := loadConfig(ctx)
+func printServerCurrent(ctx *cli.Context) error {
+	serverName, url, err := serverCurrent(ctx)
 	if err != nil {
 		return err
 	}
 
-	serverName := cf.CurrentServer
-	URL := cf.Servers[serverName].URL
-	fmt.Printf("Name: %s URL: %s\n", serverName, URL)
+	fmt.Printf("Name: %s URL: %s\n", serverName, url)
```

Comment:
nit: Should we add a second `\n` character before URL or some other form of separation?

Reply:
I don't think so. I was really just extracting some logic to the common.go, but I don't see the need to change printing format that is unrelated to the ticket since it's not objectively worse or anything.

Reply:
Ok sounds good